### DISCUSSION TITLE
SQLite to Postgres or MariaDB Migration

### DISCUSSION THREAD

**Discussion by cliqer** [directus/directus#19695] [Feature Requests]
### Summary

While in development, I have tried hard to migrate SQLite to Postgres but failed.
Had to move it into production, and now I have hit the limits of Sqlite write performance, and I desperately need to make this work as to be able to scale horizontally.

I have the Sqlite schema and managed to install it in postgre through directus migration commands, but the data needs to be translated and I have no idea how.

Can someone please help with a step-by-step guide for the latest v10.6.1 Directus?

### Motivation

---

### Detailed Design

---

### Requirements List

---

### Drawbacks

---

### Alternatives

---

### Adoption Strategy

---

**Comment by Pachat** [comment]
I'd suggest to export the SQLite data as queries: INSERT INTO...
If it needs adjustments, you may export INSERT queries from PostgreSQL (easy with phAdmin4) and adapt the queries produced by SQLite.
BTW, latest is v10.6.1 AFIK.

**Comment by DanielBiegler** [comment]
Closing this as it's not a feature request.

If you have a usage question, [Discord](https://directus.chat) is the platform we use for that. If you think you've ran into an issue while migrating your db, please [open a Bug Report](https://github.com/directus/directus/issues/new?assignees=&labels=&projects=&template=bug_report.yml). Thank you! 👍🏻
